.github/workflows/test.yml

```diff
@@ -12,6 +12,44 @@ jobs:
       - name: Run tests
         run: ./src/test
 
+  windows:
+    name: windows
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Install MinGW
+        run: |
+          choco install mingw
+          echo "C:\tools\mingw64\bin" >> $GITHUB_PATH
+
+      - name: Set CC environment variable
+        run: |
+          echo "CC=gcc" >> $GITHUB_ENV
```

Comment:
Do you actually need this? If so, please use the `env` field https://docs.github.com/en/actions/learn-github-actions/variables